src/components/global/Footer.jsx

```diff
@@ -1,8 +1,208 @@
+// Material UI Imports
+import Box from '@mui/material/Box';
+import Container from '@mui/material/Container';
+import Link from '@mui/material/Link';
+import Stack from '@mui/material/Stack';
+import Typography from '@mui/material/Typography';
+import { useTheme } from '@mui/material/styles';
+// Other Library Imports
+import dayjs from 'dayjs';
+import { FaDiscord, FaGithub, FaLinkedin, FaMeetup } from 'react-icons/fa6';
+// Image Imports
+import roseLogo from '/assets/rose_logo.png';
+import logoBlob from '/assets/logoBlob.svg';
+
+// Legal links
+const legalLinks = [
+  {
+    href: 'https://www.codepdx.org/',
+    title: 'About'
+  },
+  {
+    href: 'https://www.codepdx.org/',
+    title: 'Code of Conduct'
+  },
+  {
+    href: 'https://www.codepdx.org/',
+    title: 'Contact Us'
+  },
+  {
+    href: 'https://www.codepdx.org/',
+    target: '_blank',
+    rel: 'noopener',
+    ml: 0.5,
+    text: `©${dayjs().year()}`,
+    title: 'CODE PDX'
+  }
+];
+
+// Social media links
+const socialMediaLinks = [
+  {
+    href: 'https://github.com/codeforpdx/',
+    icon: <FaGithub size={45} />,
+    target: '_blank',
+    rel: 'noopener'
+  },
+  {
+    href: 'https://www.linkedin.com/company/code-pdx/',
+    icon: <FaLinkedin size={45} />,
+    target: '_blank',
+    rel: 'noopener'
+  },
+  {
+    href: 'https://www.meetup.com/Code-for-PDX/',
+    icon: <FaMeetup size={45} />,
+    target: '_blank',
+    rel: 'noopener'
+  },
+  {
+    href: 'https://discord.gg/x6b573et',
+    icon: <FaDiscord size={45} />,
+    target: '_blank',
+    rel: 'noopener'
+  }
+];
+
+// renders socialMediaLinks for placement in stack below
+const renderSocialLinks = socialMediaLinks.map(({ href, icon }) => (
+  <Link
+    key={href}
+    href={href}
+    target="_blank"
+    rel="noopener"
+    color="#000"
+    sx={{
+      '&:hover': {
+        color: 'secondary.main'
+      }
+    }}
+  >
+    {icon}
+  </Link>
+));
+
+const socialBlobStyle = {
+  backgroundImage: 'url(/assets/socialsBlob.svg)',
+  backgroundSize: 'cover',
+  backgroundRepeat: 'no-repeat',
+  backgroundPosition: 'center',
+  overflow: 'visible',
+  minWidth: '42%',
+  minHeight: '175px'
+};
+
+const logoBlobStyle = {
+  backgroundImage: 'url(/assets/logoBlob.svg)',
+  backgroundRepeat: 'no-repeat',
+  backgroundPosition: 'center',
+  backgroundSize: 'cover',
+  overflow: 'visible',
+  minHeight: '40vh',
```

Comment:
I don't know about these `minHeight`s and `minWidth`s...

we want the blob on the logo to be 40% of the device height and width?

Reply:
was also removed right after last push and will be out in the next one. The changes in the last push reflected flexbox changes but basically everything regarding the blobs is experimental.